windows/bleachbit_nsis_general.pot

```diff
@@ -0,0 +1,27 @@
+# English translation of bleachbit_nsis_general.pot
+# Copyright (C) 2019 Andrew Ziem
+# This file is distributed under the same license as the BleachBit package.
+# Tobias B. Besemer <Tobias.Besemer@Googlemail.com>, 2019.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: bleachbit\n"
+"Report-Msgid-Bugs-To: <ahz001+github@gmail.com>\n"
+"POT-Creation-Date: 2019-03-31 08:45+0200\n"
+"PO-Revision-Date: 2019-03-31 08:45+0200\n"
+"Last-Translator: Tobias B. Besemer <Tobias.Besemer@Googlemail.com>\n"
+"Language-Team: English <en@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "BLEACHBIT_SHELL_TITLE"
+msgstr "Shred with BleachBit"
+
+msgid "BLEACHBIT_UPGRADE_UNINSTALL"
+msgstr "${prodname} is already installed. Click 'OK' to uninstall the old version before \ upgrading, or click 'Cancel' to abort the upgrade."
```

Comment:
Does the slash \ belong in the middle here?

Reply:
I copied the string from you. Is it a linebreak in NSIS?

Reply:
OK, written in this form, it is no line-break! Also: NSIS do line-breaks by its own! So I removed the " \ "...